cli/go.mod

```diff
@@ -423,4 +423,6 @@ replace github.com/containerd/containerd => github.com/containerd/containerd v1.
 // Replace these for trivy
 replace oras.land/oras-go => oras.land/oras-go v1.1.1
 
+// /Users/idanf/go/pkg/mod/github.com/containers/image/v5@v5.19.0/sif/src.go:92:3: unknown field 'Architecture' in struct literal of type v1.Image
```

Comment:
what is this for?

Reply:
for us to remember why we put that replace